### PR TITLE
OCPBUGS-14798: Adds reference to contributing.md

### PR DESCRIPTION
**Pull request by apoorvajagtap**
Adding reference to Contributing.md for someone looking for "how to contribute/where to start".
I've tried to keep it short. Open to suggestions to make it more relatable to the community/end-users :).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # builder
 The image run by build pods to execute image building+pushing
+
+### Looking for contribution guidelines?
+Glad to have you here!<br>
```
Comment (coreydaley) [inline README.md:5]: Replace the `<br>` with two spaces instead.

Comment (apoorvajagtap) [reply README.md]: Thank you! Pushed the updated file.

**Pull request merged**